packages/react-server-website/pages/source-api.js

```diff
@@ -8,7 +8,7 @@ export default class SourceApi {
 	}
 
 	getContentType() {
-		return 'text/html';
+		return 'application/json';
```

Comment:
`ReactServerAgent` actually won't transfer `text/html` responses.

Requests to this endpoint weren't getting bundled.